articles/enforce-os-updates.md

```diff
@@ -12,10 +12,12 @@ For Apple (macOS, iOS, and iPadOS) hosts, Apple provides a [list of available OS
 
 1. Head to the **Controls** > **OS updates** tab.
 
-2. To enforce OS updates for macOS, iOS, or iPadOS, select the platform and set a **Minimum version** and **Deadline**.
+2. To enforce OS updates for enrolled macOS, iOS, or iPadOS hosts, select the platform and set a **Minimum version** and **Deadline**.
 
 3. For Windows, select **Windows** and set a **Deadline** and **Grace period**.
 
+4. *macOS only*: check "Update new hosts to latest" if you would like hosts to automatically update to the latest OS version during automatic (ADE) enrollment, regardless of the minimum version and deadline settings.
+
```

Comment:
@rachaelshaw I went back and forth about the order of these two. The Windows part feels like it lacks context if it doesn't come right after the first step, so I kept it there. But it does feel a bit weird to flip back and forth between mac and Windows. I also tried putting the "update new hosts" stuff in the second step, but it made it a lot bigger than the other two.